contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -642,6 +665,16 @@ def _execute(self, track_name_filter: str):
         logging.info("Verified that '(selection)' tabs are enabled")
 
 
```

Comment:
> In case of AutoFrameTrack we agreed with @ronaldfw that switching the option will only apply to the next session, and for that reason we need to end the session after.

Thanks for clarifying; makes sense to split this from the other parameters if the setting has a different "lifetime". Maybe add some comment here. I had the same thought as Florian about adding this to "Capture(...)".

Reply:
added :

```
    """
    Set the auto frame track setting option. This is a public method and not a parameter in Capture()
    because since the already added FrameTrack won't be removed from the current session, the E2E
    tests will need to end and start a new session before taking the capture.
    """
```